src/test/java/org/ays/tests/institution/assignmentmanagementservice/PostAssignmentTest.java

```diff
@@ -72,7 +72,7 @@ public void createAssignmentWithInvalidCountryCode(String countryCode) {
     }
 
 
-    @Test(dataProvider = "invalidLineNumberData")
+    @Test(dataProvider = "invalidLineNumberData",dataProviderClass = DataProvider.class)
```

Comment:
Kod formatında problem var. Bu tür durumlar yaşamamak için aşağıdaki dokümanı inceleyebilirsiniz;
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html